### DISCUSSION TITLE
Agenda: Brainstorming and sharing ideas for potential project directions.

### DISCUSSION THREAD

**Discussion by SomaiaElbaradey** [SomaiaElbaradey/3D-ObjectDetector#1] [Weekly meetings notes]
## Ideas Discussed 
1. Detection and Classification of Road Signs and Traffic Elements
**Contributor**: `Hassan Nawazish`
**Focus**: Detecting and classifying road signs, pedestrians, traffic signals, lanes, and vehicle types.
**Goal**: Contribute to the automotive industry by utilizing hardware resources.
**Deployment**: Can be deployed on hardware like Raspberry Pi or Jetson Nano.

2. Image Segmentation for Object Detection
**Contributor**: `Somaia`
**Focus**: Image segmentation to detect objects in pictures and separate different components from the background.
**Example**: Segregating animals, humans, or other objects based on selected categories.

3. Face Mask Detection in Elevators
**Contributor**: `Samer`
**Focus**: Detection and classification of face masks, specifically in elevators.
**Goal**: Integrate with elevator systems to generate warnings for mask violations.

4. 3D Object Detection and Classification
**Contributor**: `Rafay`
**Focus**: Detecting and classifying any 3D object, customizable according to user needs.
**Deployment**: Hardware deployment on platforms like Jetson Nano or Raspberry Pi.

5. 5D Manipulator for Object Detection and Placement
**Contributor**: `Sameed`
**Focus**: Detecting and positioning 2D/3D objects using a 5D manipulator.
**Key Features**: Extracting center and boundary points, calculating joint angles using reverse kinematics.
**Implementation**: Simulation in MATLAB for trajectory positioning, followed by deployment on hardware using Jetson Nano.

## Next Steps
Review ideas.
Explore hardware options for deployment.

--------------------------------------------------------------------------------------------------------

Friday, 20th September

**Comment by Hassannawazish** [comment]


**Reply by Hassannawazish** [reply]
can you create the tasks on github as well?
Like Jira so I create the branch according to the ticket number.?

**Reply by SomaiaElbaradey** [reply]
sure..
